gnome-initial-setup/gis-driver.c

```diff
@@ -394,11 +394,83 @@ load_vendor_conf_file (GisDriver *driver)
   priv->vendor_conf_file = g_steal_pointer (&vendor_conf_file);
 }
 
-GKeyFile *
-gis_driver_get_vendor_conf_file (GisDriver *driver)
+static void
+report_conf_error_if_needed (const gchar *group,
+                             const gchar *key,
+                             GError *error)
+{
+  if (!g_error_matches (error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND) &&
+      !g_error_matches (error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_GROUP_NOT_FOUND))
+    g_warning ("Error getting the value for key '%s' of group [%s] in the "
+               "vendor conf file: %s", group, key, error->message);
+}
+
+gboolean
+gis_driver_conf_get_boolean (GisDriver *driver,
+                             const gchar *group,
+                             const gchar *key,
+                             gboolean default_value)
+{
+  GisDriverPrivate *priv = gis_driver_get_instance_private (driver);
+
+  if (priv->vendor_conf_file) {
```

Comment:
Yeah... About that: This is not consistent throughout the code base, and I was thinking we could perhaps agree on keeping 2-space indentation but bracket indentation (or same-line brackets) for `if/else` cases. I never liked the extra indentation in the GNU style. I will write a contribution file soon clarifying which style to use in newly added code.
But this is not my sole decision, so let's see if we get an agreement...

Reply:
I tried to get data about which brace and parameter-alignment style is more common in this codebase using grep, but just when I thought I had it, I realised that `--count` was not behaving correctly in the multi-line grep case. The great thing about the Unix "everything is just lines of text" model is that it doesn't work if the text contains newlines…

Reply:
Ah, Vim can do it!

```vim
:vimgrep /\<\(if\|else\|while\|do\|switch\)\>.*\n\s\+{/ **/*.c
(1 of 76): ...
:vimgrep /\<\(if\|else\|while\|do\|switch\)\>.*\s\+{/ **/*.c
(1 of 323):
```

This is a bit misleading because there are hundreds of matches for the second style in `cc-*.c` files copied from gnome-control-center, but even so, it's not true that the brace-on-new-line style is more common (as I'd claimed).